index.html

```diff
@@ -2484,6 +2484,84 @@ <h3>serviceEndpoint</h3>
+                    <dt>Range</dt>
+                    <dd>Signature</dd>
+                  </dl>
+                  <p>
+          The following example demonstrates how a proof is expressed using
+          a JSON-LD proof with a link to an associated <code>x509Certificate</code>:
+                  </p>
+                  <pre class="example prettyprint language-jsonld">
+                    {
+                      ...
+                      "proof": {
+                        "type": "Ed25519Signature2018",
+                        "created": "2020-04-03T18:03:33Z",
+                        "x509Certificate": "MIIC/jCCAeYCCQDqa4lnWhygnDANBgkqhkiG9w0BAQsFADBBMQswCQYDVQQGEwJT
+                        RTEQMA4GA1UECgwHSXNzdWVyIDEPMA0GA1UECwwGaXNzdWVyMQ8wDQYDVQQDDAZp
+                        c3N1ZXIwHhcNMjEwNDE2MDMwMTUyWhcNMjQwMTExMDMwMTUyWjBBMQswCQYDVQQG
+                        EwJTRTEQMA4GA1UECgwHSXNzdWVyIDEPMA0GA1UECwwGaXNzdWVyMQ8wDQYDVQQD
+                        DAZpc3N1ZXIwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDYa1IJIrKr
+                        R5SHPfyOCWKuwcJHnDjTt+8cC7SLwtrfg3vazE+uRSP8cS7RitvMqK1YyC/pthJI
+                        zJlzF3OqcRQENjPfzQaD6J/55TWhwYfNj5IOVDCcZflfe+fgdl+N2mhrkIADqK3i
+                        Z5fOp5M2DN0m3XVhUllOHQ1dX6IRGCFrDV0ho2uOY+mmy3qIB3DRBLqd9pCN7ddr
+                        z+3uTRUYyuRiLdvvc7Myjq7CrNkiqr5yedMXwM9Bi3GVUIZ3L4MKgyCf7XZ0TMoX
+                        i4FVWbi4O1caKM0foglpRzGQM0IPgwQ/JR5u+rWaNx5XdowraWqU0LziznibvuFI
+                        HTpcQVmO0zNDAgMBAAEwDQYJKoZIhvcNAQELBQADggEBAAJPFM32NBdAsGnuChQ9
+                        1A66HvVwQeC/r2wJFVj3jg7h7lTYjmaHKoRP3J5UGmHo8fTbIxyJPKoXZ3qoj31Y
+                        uBFKxNaNjSrPTOUv/4injvqjkdCvQdy0iuQaNtQJSkse4t/fnQ+DMC+E3cNii8Yj
+                        0Z+T3oJL1PCg01N6uL8KtooEmnUGzIzuZ2naokgwaiEOU2CCnJOCKYxz22cxqfNm
+                        sGy1keqt6/bE64UnJRgETdqHzZTO0013ZtvDiwjGo3CZg4FBi/O722XnxbzuFIPZ
+                        WWOOb9VaH+AE4lNei6i/CDY/WcgpP0GDIj3z6lCeLtiSagiQQPYxIz2sBxC1OKVU
+                        7CE=",
+                        "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..ktzmVCTovhKWKxnMwDtMnmpUAUF1n7sqbdNmOV0TkxCTb2eHATO6iP7wqjyl_CMehHKnqgW6t9SW4q9TdbCxAg",
+                        "proofPurpose": "assertionMethod",
+                        "verificationMethod": "did:key:z6MkkUNcUWxAbxamK2Spp92mKWoXddS3tTqBKe8VTGExzX2Y#z6MkkUNcUWxAbxamK2Spp92mKWoXddS3tTqBKe8VTGExzX2Y"
+                      }
+                    }
+                  </pre>
+      </section>
+
+      <section id="x509CertificateFingerprint" about="https://w3id.org/security#x509CertificateFinger"
+      typeof="rdf:Property">
+        <h3>x509CertificateFingerprint</h3>
+        <p>
+          The x509CertificateFingerprint property is used to associate an <a href="https://tools.ietf.org/html/rfc5280">X.509 Certificate</a> with a proof via its SHA-256 based fingerprint.
```

Comment:
This property should specify the encoding format for the value. I suggest we use multibase.

Reply:
Since X.509/PKIX already specifies fingerprint formats in Ascii wouldnt it be better to simply use these?

Reply:
@David-Chadwick, if they use multibase, they are compressible in CBOR-LD.

Reply:
You should think of the customers/users. If the person already has an X.509 PKC along with a fingerprint then you should accept it as is, without requiring him to jump through formatting hoops to convert it into something that you want.

Reply:
Priority of constituencies: 1. end users, 2. devs, 3. spec/vocab authors.

I think the "users" you're referring to here are "devs". If a use case is consumption of this information by end users via QR codes (i.e., made possible by CBOR-LD), then we should favor multibase encoding because end users are the top priority. If that's not at all expected to be a common use case, then we should favor devs next -- and if they would significantly prefer another format we should go with that -- so I'd be in agreement with you then. So, what do the expected common use cases suggest we should do?

Reply:
@David-Chadwick, we can't easily compress the ASCII fingerprints in PKIX, and the driving use case for this feature is the use in vaccination certificates, which need to be highly byte compressed using multibase and CBOR-LD. You can always generate a human-readable fingerprint from the compressed data.

To give you an idea of the size difference, an ASCII SHA-256 x509 certificate fingerprint is 95 bytes, whereas a byte encoding of the same value is 32 bytes (which is 1/3rd of the size). This matters when you want to keep payload sizes below 500 bytes (which is the typical top end scanning range of most low-end QR Code scanners).

Reply:
I think compressing any VC into a QR code is a non-starter for several reasons. 1. It breaks privacy and does not allow for selective disclosure 2. It constrains the VCs contents, esp. photos are ruled out. The WHO prefers the QR code to be a pointer to a digital twin as do many other implementors.

Reply:
@dlongley. Yes I am thinking of devs (or implementors) who are used to using X.509 tools.

Reply:
> The WHO prefers the QR code to be a pointer to a digital twin as do many other implementors.

As a member of the WHO SVC working group I can assure you that this claim is completely incorrect.

Reply:
then please explain how you can reconcile your assertion with this statement from the WHO:
"Subsequent vaccine administration events logged to the same SVC paper card will be added as new entries to the signed SVC document on the SVC Registry that is referenced by the SVC paper card’s UVCI."
A static QR code containing a subset of initial event information cannot fulfil this requirement. Only a QR code that points to the digital twin can (unless you know of some other mechanism)

Reply:
Due to the confidentiality agreement signed as apart of that working group members cannot discuss details that are not apart of public releases until they are.

> Subsequent vaccine administration events logged to the same SVC paper card will be added as new entries to the signed SVC document on the SVC Registry that is referenced by the SVC paper card’s UVCI

This statement simply does not equate to your assertion of "The WHO prefers the QR code to be a pointer to a digital twin" but as clarity please refer to section 3.3.2 of RC 1.

> Release Candidate 2 of this specification will provide more detail about possible options to print barcoded content on the SVC card besides just the UVCI.

Furthermore throughout the document there are numerous references to being able to perform an offline verification check of an SVC, something that cannot be done if the information encoded in the 2D barcode is only a reference. See section 3.4.2 for instance.

Reply:
Since x509 certificates in PEM format already leverage base64url are we best to use this encoding scheme for this property too?

Reply:
> Since x509 certificates in PEM format already leverage base64url are we best to use this encoding scheme for this property too?

They use base64pad, not base64url. I expect we would default to base64pad.